### ISSUE TITLE
FixedWaveFormView crashes sometimes

### ISSUE THREAD

**Issue by AnwarShahriar** [hushed/WaveFormViewDemo#2]
There's no fixed reproducible path. Play with it in a `RecyclerView`, this will eventually happen.

```
java.lang.ArithmeticException: divide by zero
        at space.siy.waveformview.FixedWaveFormView$data$1.invokeSuspend(FixedWaveFormView.kt:61)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:238)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:594)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.access$runSafely(CoroutineScheduler.kt:60)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:742)
```

**Comment by AnwarShahriar** [comment]
Possibly because `width` can be `0` if the view tree is not populated. A `ViewTreeObserver` should fix the issue.

**Comment by AnwarShahriar** [comment]
This should be fixed with all the fixes in PR #11. Reopen it if not fixed.
